web/cypress/integration/consumer/create-consumer-with-api-breaker-plugin-form.spec.js

```diff
@@ -60,17 +59,13 @@ context('Create and delete consumer with api-breaker plugin form', () => {
     });
     cy.focused(selector.drawer).should('exist');
     cy.get(selector.disabledSwitcher).click();
-    // edit codemirror
-    cy.get(selector.codeMirror)
-      .first()
-      .then((editor) => {
-        editor[0].CodeMirror.setValue(
-          JSON.stringify({
-            key: 'test',
-          }),
-        );
-        cy.contains('button', 'Submit').click();
-      });
+
+    // edit monaco
+    cy.window().then((window) => {
+      cy.waitUntil(() => window.monacoEditor)
```

Comment:
hi @qian0817, I noticed you import a new lib `cypress-wait-until`, could you please also offer some useful references about why you choose this lib, and Introduce the risks it can pose and its usage scenarios, that would helpful for reviewers and other developers, thanks

Reply:
@liuxiran I need to wait for `window.monacoEditor` to be set which means monaco editor is initialized successfully. Can you tell me any better way to achieve this, please.

Reply:
sorry for late reply @qian0817 , actually I don't have better way  and what  I mean is that you can add note or some references to helper new comers.
anyway , thanks for your great work

Reply:
also cc  @LiteSun @guoqqqi see if there are any good suggestions

Reply:
I'm not familiar with this framework, does the monacoEditor have the ability to expose the rendered events? Like the finished event of EChars. Also, I think it is ok to use waitUntil

Reply:
@LiteSun Do you mean the onMount event?

Reply:
Hi @qian0817  You can add the assertion after 60 lines of focused 
```
    cy.get('.view-zones').should('exist');
```
This ensures that the ```monacoEditor``` is initialized successfully before manipulating the dom

Reply:
like this 
![image](https://user-images.githubusercontent.com/72343596/121813632-77666e00-cc9f-11eb-962b-fbb62ddbc476.png)

Reply:
@guoqqqi It did't run well and some of the test cases run failed.